docs/RELEASE_NOTES.md

```diff
@@ -1,5 +1,22 @@
 # Release Notes
 
+## v11.1
+
+_October 2021:_ Version 11.1
+
+11.1 is a bug-fix release.  After many changes in 11.0, there were a few problems that surfaced and with the help of reporting from the community (special shoutout to Karla Bradley at NYCET and Brendan, Daniel, and Amy at State Voices).
+
+### Bug-fixes
+* Campaign Admin: Texter manual changing/adding of assigned triggered app failure
+* Setting/Updating Twilio credentials in admin Settings failed
+* Buying numbers for an organization's message service failed
```

Comment:
If these were fixed, do we want to end the sentence with "failed?" changing it to "failure" like the note above might be best.

Outside of that minor detail, this looks great!